lib/google_drive/session.rb

```diff
@@ -418,6 +418,24 @@ def collection_by_url(url)
 
     alias folder_by_url collection_by_url
 
+    # Returns GoogleDrive::Collection with given +key+.
+    #
+    # e.g.
+    #   # https://drive.google.com/drive/u/2/folders/1rPPuzAew4tO3ORc88Vz1JscPCcnrX7-J
```

Comment:
Can you use more normal URL here as an example, not one with multi account sign in? e.g.:
https://drive.google.com/drive/folders/1rPPuzAew4tO3ORc88Vz1JscPCcnrX7-J